### PR TITLE
Add: Sansa

### PR DESCRIPTION
**Pull request by stephanflemming**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Sansa: Structural variant (SV) annotation.

Belongs to the delly suite, but uses a separate repository. There is only one sub command at the moment. Not unlikely that more will be added in future releases.

GitHub: https://github.com/dellytools/sansa
(My) parameter overview: https://docs.google.com/spreadsheets/d/1_JFVghlnHyYNWoA3ao-t0LXzH-8XwGm_Bsq0YK6XNoU/edit?usp=sharing

### CODE REVIEW

**Comment by stephanflemming** [comment]
There is only one sub command at the moment. Not unlikely that more will be added in future releases (like delly). That's why I implemented a suite.

File: tools/sansa/macros.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            <citation type="doi">10.1093/bioinformatics/bts378</citation>
+        </citations>
+    </xml>
+
+    <!-- input -->
+
+    <xml name="fg">
+        <section name="fg" title="Feature and gene annotation options" expanded="true">
+            <param argument="--gtf" type="data" format="gtf,gff3" label="Select file"/>
+            <param argument="--id" type="text" value="gene_name" label="Set GTF/GFF3 attribute">
```
Comment (bgruening) [inline tools/sansa/macros.xml:24]: type=select ?

**Review by bgruening — COMMENTED** [review]
Top, just one small question!

**Comment by bgruening** [comment]
This is the failure: 
```
Output out_anno:  different than expected
Expected file size was 2408, actual file size was 57072 (difference of 0 accepted)

```

File: tools/sansa/macros.xml (hunk 1 above)
Comment (stephanflemming) [reply tools/sansa/macros.xml:24]: I took the "preselected" options for --id and --feature from https://github.com/dellytools/sansa ... I am not sure if other values are useful here, but as far as I understood the GTF/GFF3 file type allows many more. With type="text" the selection is not limited to values that are mentioned in the tutorial. But I am not even sure if other values are even useful here.

**Comment by stephanflemming** [comment]
> This is the failure:
> 
> ```
> Output out_anno:  different than expected
> Expected file size was 2408, actual file size was 57072 (difference of 0 accepted)
> ```

Ops.

**Review by bgruening — APPROVED** [review]
Pretty cool! Thanks!

**Pull request merged**